hack/dockerfile/etc/docker/daemon.json

```diff
@@ -3,5 +3,8 @@
 		"crun": {
 			"path": "/usr/local/bin/crun"
 		}
+	},
+	"features": {
+		"containerd-snapshotter": false
 	}
```

Comment:
DOH! Erm.. wondering if this makes things, erm, easier, or not. I was used to using the `--feature` option to enable/disable in my dev-shell, but forgot that this will now cause an error;

```bash
dockerd --feature containerd.snapshotter --debug
unable to configure the Docker daemon with file /etc/docker/daemon.json: the following directives are specified both as a flag and in the configuration file: features: (from flag: map[containerd.snapshotter:true], from file: map[containerd-snapshotter:false])
```